tox.ini

```diff
@@ -79,6 +79,7 @@ commands =
 changedir = {toxinidir}
 deps = -rrequirements/docs.txt
 commands =
+    python -m pip install .
```

Comment:
I suggested this pip install in another comment, however I think it may not be strictly necessary, since tox installs the package dependencies as part of it's environment setup. And merlin-core is specified in the package install requires. 

I tried running this `docs` and `docs-multi` locally without this and both seem to produce the API docs with content, so might be some other reason they didn't populate

Reply:
hmm. ok, yeah then I'm stumped on what the problem would have been.